### PR TITLE
BP-642 add entry-button to dialog

### PR DESCRIPTION
**Pull request by avanwaas**

### CODE REVIEW

File: libs/entry-components/dialog/dialogs/entry-dialog.component.html [hunk 1]
```diff
@@ -1,33 +1,19 @@
...
-        <h1 class="title">{{ title }}</h1>
+<h1 mat-dialog-title>{{ title }}
+    <button mat-icon-button mat-dialog-close aria-label="Close dialog" *ngIf="!hideClose">
+        <mat-icon>close</mat-icon>
+      </button>
+</h1>
+<div mat-dialog-content>
+    <ng-content></ng-content>
+</div>
+<div mat-dialog-actions [align]="buttonsAlignment">
```
Comment (avanwaas) [inline libs/entry-components/dialog/dialogs/entry-dialog.component.html:9]: add ngIf here checking hideButtons?

File: libs/entry-components/dialog/dialogs/entry-dialog.component.html [hunk 2]
```diff
@@ -1,33 +1,19 @@
...
-    <div class="dialog-header">
-        <h1 class="title">{{ title }}</h1>
+<h1 mat-dialog-title>{{ title }}
+    <button mat-icon-button mat-dialog-close aria-label="Close dialog" *ngIf="!hideClose">
+        <mat-icon>close</mat-icon>
+      </button>
+</h1>
+<div mat-dialog-content>
+    <ng-content></ng-content>
+</div>
```
Comment (typhoon41) [inline libs/entry-components/dialog/dialogs/entry-dialog.component.html:8]: How does this new alignment work? This [align] thingy comes from angular material?

File: libs/entry-components/dialog/dialogs/entry-dialog.component.html [hunk 3]
```diff
@@ -1,33 +1,19 @@
-<div class="entry-dialog dialog-form">
-    <div class="dialog-header">
-        <h1 class="title">{{ title }}</h1>
+<h1 mat-dialog-title>{{ title }}
+    <button mat-icon-button mat-dialog-close aria-label="Close dialog" *ngIf="!hideClose">
```
Comment (typhoon41) [inline libs/entry-components/dialog/dialogs/entry-dialog.component.html:2]: We still need to prettify this, cover edge cases, i.e. small dialog, long title, adequate button positioning etc.
Make a list and give it to Svetlana.

**Comment by avanwaas** [comment]
changes applied to other branch

**Pull request closed without merging**